sites/fast-website/src/app/data/framework.data.ts

```diff
@@ -29,7 +29,7 @@ export const frameworkContentPlacementData: FrameworkContentPlacementData[] = [
     },
     {
         body:
-            "While the React team works to fully support custom elements in v17, we provide integration support for v16.x.x.",
+            "While we are eagerly awaiting React to fully support custom elements, we provide integration support for v16.x.x.",
```

Comment:
```suggestion
			"We are eagerly anticipating the arrival of full custom elements support in React. In the meantime, integration support is available for React v16"
```

Reply:
Aw I just missed it. If you want, feel free to open a PR with the updates!